code/modules/paperwork/fax.dm

```diff
@@ -48,6 +50,17 @@ GLOBAL_VAR_INIT(nt_fax_department, pick("NT HR Department", "NT Legal Department
 		/obj/item/holochip,
 		/obj/item/card,
 		/obj/item/folder/biscuit,
+		//MONKESTATION EDIT START, also edits icons/obj/fax.dmi
+		/obj/item/clothing/head/mob_holder, //if i could it'd just be mothroaches but they ALL use this.
```

Comment:
me when I fax an amongus person to centcom

Reply:
Checks are later down the code specifically to prevent faxing people into cc, should work as it did last i tested